### DISCUSSION TITLE
(Confirmed) Overdumping some 'Intellivision' Roms

### DISCUSSION THREAD

**Discussion by 7ank0v1c** [sanni/cartreader#830] [Coding Corner]
(Edit: See below comment)

Heya, so below are images of the game I am trying to dump, and a hash comparison with a (apparent), correctly dumped rom. As you can see all reads fine until the last section where it seems bits are either missing, reading different or extra….

I actually have no idea about this stuff, so it could be something else. But wanted help and to potentially raise an issue.

I have cleaned pins over and over and am certain it isn’t the pins. I have also successfully dumped another cart with same cart reader and adapter. Plus hundreds of other carts for different systems so that rules out the actual cart reader (possibly not the intellivision adapter?)

![IMG_8113](https://github.com/sanni/cartreader/assets/40895945/7951c04d-9dc3-4585-b2f0-827a02ecf58b)
![IMG_8108](https://github.com/sanni/cartreader/assets/40895945/ebe9d4c9-6f80-4a3c-a264-57df6119222d)
![IMG_8109](https://github.com/sanni/cartreader/assets/40895945/bb8ca6a2-3b9b-4ac3-bfe2-e79021e29dfe)
![IMG_8110](https://github.com/sanni/cartreader/assets/40895945/c58ac125-4064-414d-92a1-ca29b2072f3e)
![IMG_8111](https://github.com/sanni/cartreader/assets/40895945/bcc834d4-0292-4845-b145-f9f41f6bba78)
![IMG_8112](https://github.com/sanni/cartreader/assets/40895945/dfe0a862-9aaf-4ba3-85b9-6fa1a782b44f)

**Comment by 7ank0v1c** [comment]
So after a bit more research and discussion on discord with the no-intro crew, it seems the sanni is overdumping some roms. This game being one of them.

Here is the source for anyone who is capable of fixing things for the sanni (@sanni possibly?):

https://forum.no-intro.org/viewtopic.php?p=34242&hilit=intellivision#p34242

**Reply by sakman55** [reply]
I'll check in with psychofox11 to see what needs to be done.
